### ISSUE TITLE
7.13.11. Screen Name/Title: API documentation - Open API Specification - Ambiguous Link Text

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2297]
Components: 
Observation: 
This issue is observed throughout GC Notify.
Links like “/v2/notifications/sms” or “/v2/templates” are used as link text.
Screen reader users don’t get meaningful context of what the link does.

(Attachment: 7.13.11. Ambiguous Link Text.png)

Recommendation: 
Use descriptive link text that makes sense out of context (e.g., avoid “Read more”).
If space is limited, use aria-label or visually hidden text to add context.
Example:
 `<a href="report.pdf" aria-label="Download 2025 Accessibility Report">Download</a>`

**Comment by YedidaZalik** [comment]
@amazingphilippe  I was working on putting the WCAG requirement for this into plain language. This is what the report cites as the WCAG requirement:
"WCAG 2.1 Success Criterion: Level A [2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/WCAG21/#link-purpose-in-context)"

WCAG says:

"The [purpose of each link](https://www.w3.org/TR/WCAG21/#dfn-purpose-of-each-link) can be determined from the link text alone or from the link text together with its [programmatically determined link context](https://www.w3.org/TR/WCAG21/#dfn-programmatically-determined-link-context), except where the purpose of the link would be [ambiguous to users in general](https://www.w3.org/TR/WCAG21/#dfn-ambiguous-to-users-in-general)."

WCAG gives this definition and example of "ambiguous to users in general"
ambiguous to users in general
"the purpose cannot be determined from the link and all information of the web page presented to the user simultaneously with the link (i.e., readers without disabilities would not know what a link would do until they activated it)

Example
The word guava in the following sentence "One of the notable exports is guava" is a link. The link could lead to a definition of guava, a chart listing the quantity of guava exported or a photograph of people harvesting guava. Until the link is activated, all readers are unsure and the person with a disability is not at any disadvantage."


Is this issue covered by "ambiguous to users in general" and if so does that mean it's not an issue and we do not need to add it to the A11y statement?

**Comment by YedidaZalik** [comment]
Disussed with Phil : It's for a developer audience, developers will understand even if they're using a screen reader and it's ambiguous for both sighted and non-sighted users
So it is "ambiguous to users in general" and therefore not an issue under 2.4.4 and we do not need to add it to the A11y statement
